src/bors/handlers/review.rs

```diff
@@ -1423,4 +1426,35 @@ merge_queue_enabled = true
             })
             .await;
     }
+
+    #[sqlx::test]
+    async fn unapprove_running_auto_build_updates_check_run(pool: sqlx::PgPool) {
+        BorsBuilder::new(pool)
+            .github(GitHubState::default().with_default_config(
+                r#"
+merge_queue_enabled = true
+"#,
+            ))
+            .run_test(async |tester: &mut BorsTester| {
+                tester.post_comment("@bors r+").await?;
+                tester.expect_comments(1).await;
+                tester.process_merge_queue().await;
+                tester.expect_comments(1).await;
+                tester
+                    .wait_for_default_pr(|pr| pr.auto_build.is_some())
+                    .await?;
+                tester.workflow_start(tester.auto_branch()).await?;
+                tester.post_comment("@bors r-").await?;
+                tester.expect_comments(2).await;
```

Comment:
This is a specific error I think, so let's check its content. I also wonder why it's two comments, shouldn't it be just one? :thinking: We should ideally combine the messages in to one.